pkg/ingester/ingester.go

```diff
@@ -953,7 +968,9 @@ type ctxKey int
 var pushReqCtxKey ctxKey = 1
 
 type pushRequestState struct {
-	requestSize int64
+	requestSize     int64
+	requestDuration time.Duration
+	err             error
```

Comment:
If you strongly prefer it to be `pushErr` I can change it. 
But since the type name `pushRequestState` already says that it is push-related, maybe having the push prefix in the error name becomes redundant (IMO it makes no sense to have other types of errors there, WDYT?). 
`pushRequestState.err` vs. `pushRequestState.pushErr`